data/quickstarts/openvino-inference-notebook.yaml

```diff
@@ -0,0 +1,50 @@
+kind: ConsoleQuickStart
+metadata:
+  name: openvino-inference-notebook
+spec:
+  displayName: Optimized Inference Notebook
+  durationMinutes: 5
+  icon: 'images/openvino.svg'
+  description: This quick start details how to use the OpenVINO toolkit.
+  introduction: |-
+    ### This quick start details how to quantize a model and use the result for inference from a notebook using the OpenVINO toolkit.
+
+The OpenVINO toolkit model optimizer can create lower precision intermediate representations of machine learning models.
+
+Using the resulting intermediate representations can reduce the time required to perform an inference operation while maintaining acceptable accuracy.
+
+This quick start will walk through how to optimize a ONNX computer vision model using the OpenVINO model optimizer, and use the resulting intermediate representation to perform hardware accelerated inference in-notebook.
```

Comment:
This needs proper indentation or the yaml fails to load.

Reply:
better?